client/scss/components/_promo.scss

```diff
@@ -25,7 +25,7 @@
   }
 }
 
-@include image-zoom ($interactionElement: '.promo', $containerElement: '.promo__image-container', $zoomElement: '.image');
+@include image-zoom ($interactionElement: '.promo', $containerElement: '.promo__image-container:not(.promo__image-container--constrained)', $zoomElement: '.image:not(.promo__image-container--constrained .image)');
```

Comment:
This is getting dangerous, excluding things like this really makes me think we should just add the class to the elements we want it on rather than a "all, oh yes, but except this one...".